packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -131,6 +139,28 @@ contract ChugSplashManager is Owned {
         string targetName
     );
 
+    /**
+     * @notice Emitted when a bundle is claimed by an executor.
+     *
+     * @param bundleId     ID of the bundle that was claimed.
+     * @param executor     Address of the executor that claimed the bundle ID for the project.
+     * @param prevExecutor Address of the executor that previously claimed the bundle ID for the
+     *                     project. This is address(0) if there was no previous executor. If there
+     *                     was a previous executor, then this event signifies that the previous
+     *                     executor's bond has been forfeited to the ChugSplashManager, which the
+     *                     project owner can claim with a call to `withdrawETH`.
+     */
+    event BundleClaimed(bytes32 indexed bundleId, address indexed executor, address prevExecutor);
```

Comment:
I think we can get rid of the `prevExecutor` field and just use the existence of more than one `BundleClaimed` event to signify that the bundle wasn't executed in time. Slightly cleans up the code too.